neurodsp/filt.py

```diff
@@ -546,6 +546,37 @@ def compute_nyquist(fs):
 
     return fs / 2.
 
+
+def infer_passtype(fc):
```

Comment:
I poked around and it looks like only `timefrequency` uses infer_passtype. Should filter also use it, or is that unnecessary since the filter_type is specified?

Reply:
As currently set up, filter doesn't really need to use it, I think, and yeh, only `timefrequency` actually uses it (which is where this code originally lived). 

I moved it to filter, partly because it seems like a filter related function - but also since in future other things *could* use it (for example, perhaps when passing in a range to filter aperiodic stuff in sim - so I thought it might get used more broadly when we do some 1.0 related updates.

Reply:
sounds good